src/HonzaBotner/Startup.cs

```diff
@@ -97,23 +107,52 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                     c.SwaggerEndpoint("/swagger/v1/swagger.json", "HonzaBotner v1");
                     c.RoutePrefix = string.Empty;
                 });
+                app.UseHttpsRedirection();
+                app.UseHangfireDashboard();
             }
             else
             {
                 UpdateDatabase(app);
+                SetupDashboard(app);
                 app.UseReverseProxyHttpsEnforcer();
             }
 
-            app.UseHttpsRedirection();
```

Comment:
Why remove this?

Reply:
https://github.com/fit-ctu-discord/honza-botner/compare/worker#diff-a4af12c288529cc8573098b3634a9596ee5eaf6c692d7835e9eb9a471c7829cfL105

This looks like it was removed in the worker branch I merged in